lms/templates/discussion/_filter_dropdown.html

```diff
@@ -34,19 +35,19 @@
     </li>
 </%def>
 
-<div class="forum-nav-browse-menu-wrapper" style="display: none">
+<div class="forum-nav-browse-menu-wrapper" style="display: none" role="combobox" aria-label="${_("Discussion thread list")}" aria-expanded="true" aria-owns="discussion_topics_listbox" aria-haspopup="listbox">
```

Comment:
@andy-armstrong I'm not clear what element the label is for: is it the entire dropdown menu? 
It's difficult that the list can be either topics or posts. Would something like "discussion dropdown" or "discussion list" be useful or too generic @cptvitamin?